cmd/internal/flowmode/cmd_run.go

```diff
@@ -450,3 +455,32 @@ func splitPeers(s, sep string) []string {
 	}
 	return strings.Split(s, sep)
 }
+
+func setMutexBlockProfiling(l log.Logger) {
+	mutexProfileFraction := os.Getenv("MUTEX_PROFILING_PERCENT")
```

Comment:
nit: feels weird to call these variables *Fraction when they're percentages and made into fractions later.